decentraland/unity-explorer
# Feat: camera reel - passport integration

## What does this PR change?



Fix #2961

Implements the "public" reel gallery in player's passport by adding a new "Photos" tab where every player's reel flagged as public is visible to other players.
Also adds analytics for reel interactions and prepares the gallery for places.

## How to test the changes?



1. Launch the explorer with the CLI arg `camera-reel`, `dclenv zone` and `debug` flags
2. Open the passport of a player and check that every reel flagged as `public` is visible and not editable
3. The passport can be either another player's or your own
4. By clicking any reel, the photo detail popup gets opened

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
